documentation/docs/strategies/advanced-mm/order-optimization.md

```diff
@@ -48,21 +48,21 @@ This allows users to ignore dust orders specified in base currency amount. As sh
 - bid_order_optimization_depth: 0
 ```
 
-![jump_orders_3](/assets/img/jump_orders5.png)
```

Comment:
Please delete jump_orders5.png file if not used in other sections of the page to avoid clutter in the `img` directory.